### DISCUSSION TITLE
v2.0.0-rc1 upgrade issue.

### DISCUSSION THREAD

**Discussion by shivakumar12** [argoproj/argo-cd#5908] [Q&A]
Hi, We’re facing issue w.r.t dex unable to login via LDAP after upgrading argocd version to v2.0.0-rc1. Getting the following error message when try to login via LDAP.
502 Bad Gateway
 unknown app state zZZcSUIYue
When we verify the dex pod logs authentication was successful.
Same configuration is working fine in version (v1.8.7).
Is it bug? or do we need make any changes to make it work. Please suggest.
Also is it recommended version(v2.0.0-rc1) use in production system?

**Comment by mayzhang2000** [comment]
When getting `unknown app state zZZcSUIYue`, restarting redis fixes the issue.

More info: https://github.com/argoproj/argo-cd/issues/5873
